City-of-Pori/pori-kada
# PORI-50: Frontpage slideshow.

Frontpage slideshow.

To test:
1. drush updb -y && drush fra -y && drush cc all
2. add some liftups to carousel
3. go to frontpage and confirm that liftups are displayed in slideshow.

NOTICE: Make sure liftup text is trimmed after certain length. If not then check the top carousel promoted content view that body field is trimmed. For whatever reason drush does not revert features properly.
